src/Microsoft.ML.Data/Transforms/Normalizer.cs

```diff
@@ -109,7 +114,11 @@ internal enum NormalizationMode
             /// <summary>
             /// Bucketize and then rescale to between -1 and 1. Calculates bins based on correlation with the Label column.
             /// </summary>
-            SupervisedBinning = 4
+            SupervisedBinning = 4,
+            /// <summary>
+            /// Centers the data around 0 and then scales based on the data range and the quantile min and max values provided.
+            /// </summary>
```

Comment:
Should this read: "Centers the data around 0 optionally and..."

Reply:
Added.